src/calculateRentalCost.js

```diff
@@ -29,6 +29,22 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const dailyPrice = 40;
+  const longTermPeriod = 7;
+  const longTermDiscount = 50;
+  const shortTermPeriod = 3;
+  const shortTermDiscount = 20;
+  let totalCost = dailyPrice * days;
+
+  if (days >= shortTermPeriod) {
+    totalCost -= shortTermDiscount;
+
+    if (days >= longTermPeriod) {
+      totalCost -= (longTermDiscount - shortTermDiscount);
```

Comment:
I think need use construction 'else if' instead of another 'if'

Reply:
we need second `if` only if first condition is true, that`s why i didn`t use `else if`

Reply:
Okay, maybe you are right)